e2e_tests/tests/nightly/test_convergence.py

```diff
@@ -161,8 +161,8 @@ def test_unets_tf_keras_accuracy(client: _client.Determined) -> None:
 
 @pytest.mark.nightly
 def test_gbt_titanic_estimator_accuracy(client: _client.Determined) -> None:
-    import tensorflow as tf
-    from packaging import version
+    import tensorflow as tf  # noqa: I2000
+    from packaging import version  # noqa: I2000
```

Comment:
I don't think either of these `noqa`s are used. I _believe_ `version` is a module, and the `as` rule makes sure that the alias is two letters long or more.

Reply:
I2000 is a different error, `Imports are only allowed on module level.`. which makes sense to enforce until the user is doing it intentionally.